frontend/src/views/components/LanguageSwitcher/LanguageSwitcher.component.tsx

```diff
@@ -3,19 +3,20 @@ import React, { useMemo } from 'react';
 
 import { tx } from '@transifex/native';
 import { useLanguages, useLocale } from '@transifex/react';
-import { useRouterParams } from 'utilities';
 import cx from 'classnames';
 
-const AVAILABLE_LANGUAGES = ['en', 'fr', 'es', 'zh', 'pt', 'ru'];
+const AVAILABLE_LANGUAGES = ['en', 'fr', 'es', 'zh-CN', 'pt-BR', 'ru'];
```

Comment:
Look at this HeCo Invest example: https://github.com/Vizzuality/heco-invest/blob/develop/frontend/locales.config.json.

We stored the locales in a separate top-level JSON and we imported it in the components and `next.config.js`. It avoids defining the locales in multiple places. It's more future-proof.